src/test/java/gumtree/spoon/AstComparatorTest.java

```diff
@@ -1,3 +1,21 @@
+/*
+/* *****************************************************************************
+ * Copyright 2016 Matias Martinez
+ * Copyright (c) 2022, Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * *****************************************************************************/
```

Comment:
Matias' name was added only because it already exists as a copyright header in one of the test files. Can change it to whatever you feel best represents the work done.

As to why the headers were added: it is a requirement set by Oracle for this kind of contribution.